File: app/src/main/java/com/prathamesh/matrimonyapp/Model/ChatUser.java
package com.prathamesh.matrimonyapp.Model;

public class ChatUser {
    private String ImageUrl;
    private String name;
    private String userId;
    private String mORr;

    public ChatUser(){

    }

    public String getImageUrl() {
        return ImageUrl;
    }

    public void setImageUrl(String imageUrl) {
        ImageUrl = imageUrl;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getmORr() {
        return mORr;
    }

    public void setmORr(String mORr) {
        this.mORr = mORr;
    }

}

File: app/src/main/java/com/prathamesh/matrimonyapp/model/ChatUser.kt
package com.prathamesh.matrimonyapp.model

class ChatUser {
    var imageUrl: String? = null
    var name: String? = null
    var userId: String? = null
    private var mORr: String? = null
    fun getmORr(): String? {
        return mORr
    }

    fun setmORr(mORr: String?) {
        this.mORr = mORr
    }
}